lms/templates/instructor/instructor_dashboard_2/executive_summary.html

```diff
@@ -0,0 +1,114 @@
+<%! from django.utils.translation import ugettext as _ %>
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Edx Executive Summary</title>
+<style type="text/css">
+body {
+	font-family: Arial, Helvetica, sans-serif;
+	font-size:14px;
+	line-height:22px;
+	margin: 10px;
+}
+.box-bg {
+	background:#f1f1f1;
+	padding:10px;
+}
+th {
+	padding:5px;
+	background:#ccc;
+}
+h2 {
+	margin-top:0
+}
+</style>
+</head>
+
+<body>
+
+<table width="650" border="0" cellspacing="5" cellpadding="5">
+	<tr>
+		<td align="left" valign="top"><img src="https://d37djvu3ytnwxt.cloudfront.net/static/mit-prof-ed/images/MITPE-Digital-Programs-Logo.png" alt="logo" /></td>
```

Comment:
Sorry, thanks for the extra effort, but we can't hard code a branded logo here. Let's remove this for now.